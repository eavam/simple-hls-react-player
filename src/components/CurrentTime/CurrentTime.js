// @flow
import React from 'react';
import styled from 'react-emotion';
import withConnect from '../../hocs/withConnect';

const Root = styled('div')`
  font-size: 19px;
  color: #fff;
`;

export const CurrentTime = ({ time }: { time?: string }) => <Root>{time}</Root>;

CurrentTime.defaultProps = {
  time: '00:00',
};

export default withConnect(['time'])(CurrentTime);
