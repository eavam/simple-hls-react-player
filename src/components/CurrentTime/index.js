import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const CurrentTime = ({ time }) => <Root>{time}</Root>;

CurrentTime.propTypes = {
  time: PropTypes.string
};

CurrentTime.defaultProps = {
  time: '00:00'
};

const Root = styled.div`
  font-size: 19px;
  color: #fff;
`;

export default CurrentTime;
