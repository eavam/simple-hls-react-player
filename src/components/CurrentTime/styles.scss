.current-time {
  font-size: 19px;
  color: #fff;
}