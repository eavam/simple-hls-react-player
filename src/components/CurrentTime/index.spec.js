import React from 'react'
import renderer from 'react-test-renderer'
import CurrentTime from './index.js'

test('Current time ', () => {

  const props = {
    time: '03:23'
  }

  const component = renderer.create(
    <CurrentTime {...props} />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()

})