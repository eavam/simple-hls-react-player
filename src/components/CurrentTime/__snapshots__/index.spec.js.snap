// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Current time  1`] = `
<div
  className="sc-bdVaJa hcxCtn"
>
  03:23
</div>
`;
